Index.html

```diff
@@ -1,14 +1,14 @@
 <!DOCTYPE html>
-<script src="https://code.jquery.com/jquery-1.11.3.min.js"></script>
 <html>
-    <head><script>
-        $(function(){
-            $("#includeHtml").load("Navbar.html");
-        });
-    </script></head>
-  <link rel="stylesheet" href="Style.css" />
-  <div id="includeHtml"></div>
-  <p class="Hello">Hello!</p>
-
-  <h1 class="Heading">I am Anupama Jha</h1>
+  <head>
```

Comment:
include meta tags, they help browser and web crawlers understand your webpage more efficiently, 

visit: https://www.w3schools.com/tags/tag_meta.asp,